Block/ProductLabel/ProductLabel.php

```diff
@@ -139,10 +208,17 @@ public function getProductLabels()
      *
      * @return string
      */
-    public function getImageUrl($imageName) {
+    public function getImageUrl($imageName)
+    {
        return $this->imageHelper->getBaseUrl().'/'.$imageName;
     }
 
+
```

Comment:
If this method is useless, it can probably be removed :)

Reply:
Done